Mtneer/mushu
# Add category

# Description
> As a user I would like to be able to create a new category so that I can give myself the ability to better classify their posts.
> 
> **Given** an admin is on the Category list page
> **When** they select the `Create Category` button
> **Then** they should be directed to a form in which they can enter a new category name
> 
> **Given** an admin has entered a Category name
> **When** they click the `Save` button
> **Then** a new category should be saved to the database
> **And** the admin should be directed to the Category list page

## Type of change
Please delete options that are not relevant.

# Testing Instructions
- pull down new branch with git fetch -a and checkout to git checkout add-category
- start the react app and web api
- login to the app using admin@admin.com and password1
- click on the category tab
- select the "Create Category" tab
- type in whatever you want to test that Categories can be added to the database.
- click the save button
- verify that you are redirected to the list of categories and that your newly added category appears in the list alphabetically.

# Checklist:
